### ISSUE TITLE
Github related octicons do not work

### ISSUE THREAD

**Issue by PirateSeal** [retypeapp/retype#517]
Hi,

GitHub related octicons do not work correctly. I'm trying to use [this one](https://primer.github.io/octicons/mark-github-16) but i'm getting this :
![image](https://user-images.githubusercontent.com/33653999/227554030-855b64f6-5a34-4bc7-a198-75a9478e97a8.png) 

I used this in my page

```markdown
===
==- :icon-comment-discussion: My social networks
[!ref target="blank" icon="mark-github" text="GitHub"](https://github.com/pirateseal)
===
```

**Comment by geoffreymcgill** [comment]
Looks like it is all the icons that only exist in the `16px` size:

```
accessibility
accessibility-inset
apps
cache
copilot-error
copilot-warning
device-camera
diff-added
diff-ignored
diff-modified
diff-removed
diff-renamed
ellipsis
feed-discussion
feed-forked
feed-heart
feed-merged
feed-person
feed-repo
feed-rocket
feed-star
feed-tag
feed-trophy
file-added
file-badge
file-directory-open-fill
file-moved
file-removed
fiscal-host
id-badge
key-asterisk
logo-gist
logo-github
mark-github
markdown
meter
paintbrush
repo-clone
repo-deleted
repo-pull
sliders
sparkle-fill
tab-external
three-bars
webhook
```

Here's a test demonstrating that the icons work in the Button component, but not the Ref component:

```md
Icon | Button | Ref
--- | --- | ---
| `accessibility` | [!button icon="accessibility"] | [!ref icon="accessibility"](https://example.com) |
| `accessibility-inset` | [!button icon="accessibility-inset"] | [!ref icon="accessibility-inset"](https://example.com) |
| `apps` | [!button icon="apps"] | [!ref icon="apps"](https://example.com) |
| `cache` | [!button icon="cache"] | [!ref icon="cache"](https://example.com) |
| `copilot-error` | [!button icon="copilot-error"] | [!ref icon="copilot-error"](https://example.com) |
| `copilot-warning` | [!button icon="copilot-warning"] | [!ref icon="copilot-warning"](https://example.com) |
| `device-camera` | [!button icon="device-camera"] | [!ref icon="device-camera"](https://example.com) |
| `diff-added` | [!button icon="diff-added"] | [!ref icon="diff-added"](https://example.com) |
| `diff-ignored` | [!button icon="diff-ignored"] | [!ref icon="diff-ignored"](https://example.com) |
| `diff-modified` | [!button icon="diff-modified"] | [!ref icon="diff-modified"](https://example.com) |
| `diff-removed` | [!button icon="diff-removed"] | [!ref icon="diff-removed"](https://example.com) |
| `diff-renamed` | [!button icon="diff-renamed"] | [!ref icon="diff-renamed"](https://example.com) |
| `ellipsis` | [!button icon="ellipsis"] | [!ref icon="ellipsis"](https://example.com) |
| `feed-discussion` | [!button icon="feed-discussion"] | [!ref icon="feed-discussion"](https://example.com) |
| `feed-forked` | [!button icon="feed-forked"] | [!ref icon="feed-forked"](https://example.com) |
| `feed-heart` | [!button icon="feed-heart"] | [!ref icon="feed-heart"](https://example.com) |
| `feed-merged` | [!button icon="feed-merged"] | [!ref icon="feed-merged"](https://example.com) |
| `feed-person` | [!button icon="feed-person"] | [!ref icon="feed-person"](https://example.com) |
| `feed-repo` | [!button icon="feed-repo"] | [!ref icon="feed-repo"](https://example.com) |
| `feed-rocket` | [!button icon="feed-rocket"] | [!ref icon="feed-rocket"](https://example.com) |
| `feed-star` | [!button icon="feed-star"] | [!ref icon="feed-star"](https://example.com) |
| `feed-tag` | [!button icon="feed-tag"] | [!ref icon="feed-tag"](https://example.com) |
| `feed-trophy` | [!button icon="feed-trophy"] | [!ref icon="feed-trophy"](https://example.com) |
| `file-added` | [!button icon="file-added"] | [!ref icon="file-added"](https://example.com) |
| `file-badge` | [!button icon="file-badge"] | [!ref icon="file-badge"](https://example.com) |
| `file-directory-open-fill` | [!button icon="file-directory-open-fill"] | [!ref icon="file-directory-open-fill"](https://example.com) |
| `file-moved` | [!button icon="file-moved"] | [!ref icon="file-moved"](https://example.com) |
| `file-removed` | [!button icon="file-removed"] | [!ref icon="file-removed"](https://example.com) |
| `fiscal-host` | [!button icon="fiscal-host"] | [!ref icon="fiscal-host"](https://example.com) |
| `id-badge` | [!button icon="id-badge"] | [!ref icon="id-badge"](https://example.com) |
| `key-asterisk` | [!button icon="key-asterisk"] | [!ref icon="key-asterisk"](https://example.com) |
| `logo-gist` | [!button icon="logo-gist"] | [!ref icon="logo-gist"](https://example.com) |
| `logo-github` | [!button icon="logo-github"] | [!ref icon="logo-github"](https://example.com) |
| `mark-github` | [!button icon="mark-github"] | [!ref icon="mark-github"](https://example.com) |
| `markdown` | [!button icon="markdown"] | [!ref icon="markdown"](https://example.com) |
| `meter` | [!button icon="meter"] | [!ref icon="meter"](https://example.com) |
| `paintbrush` | [!button icon="paintbrush"] | [!ref icon="paintbrush"](https://example.com) |
| `repo-clone` | [!button icon="repo-clone"] | [!ref icon="repo-clone"](https://example.com) |
| `repo-deleted` | [!button icon="repo-deleted"] | [!ref icon="repo-deleted"](https://example.com) |
| `repo-pull` | [!button icon="repo-pull"] | [!ref icon="repo-pull"](https://example.com) |
| `sliders` | [!button icon="sliders"] | [!ref icon="sliders"](https://example.com) |
| `sparkle-fill` | [!button icon="sparkle-fill"] | [!ref icon="sparkle-fill"](https://example.com) |
| `tab-external` | [!button icon="tab-external"] | [!ref icon="tab-external"](https://example.com) |
| `three-bars` | [!button icon="three-bars"] | [!ref icon="three-bars"](https://example.com) |
| `webhook` | [!button icon="webhook"] | [!ref icon="webhook"](https://example.com) |
```

<img width="781" alt="Screenshot 2023-03-24 at 10 19 30 AM" src="https://user-images.githubusercontent.com/62210/227582864-3b37cc49-da35-458d-af49-1e0d548e329c.png">

**Comment by geoffreymcgill** [comment]
This defect has been fixed and will be available in the upcoming Retype v3.0 release. 

Thanks for reporting the issue!
